release.py

```diff
@@ -240,11 +243,16 @@ def print_release_notes(notes: str):
 def input_title_and_summary(release: github.GitRelease.GitRelease) -> tuple[str, str]:
     """Ask user to input the release title and summary."""
     logger.info(f'The automatically generated title is: {release.title}')
-    title = input('Enter release title, press Enter to keep the auto-generated title:\n> ').strip()
+    title = input(
+        '\033[1mEnter release title, press Enter to keep the auto-generated title:\n> \033[0m'
+    ).strip()
     if not title:
         title = release.title
 
-    print("\nEnter release summary (multi-line supported; type '.' on a new line to finish):")
+    print(
+        '\n\033[1mEnter release summary (multi-line supported; type "." on a new line '
+        'to finish):\n> \033[0m'
+    )
```

Comment:
This is the one case where I think the leading `>` is odd, because it's (probably) multi-line text. It's probably best to be consistent, though.

I wonder if in a future PR we could do a subprocess to `EDITOR` here (maybe even passing in the list of changes in the initial file and trimming them out in the response, a little like interactive rebase).